plugin/trino-iceberg/pom.xml

```diff
@@ -341,6 +341,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
I'm thinking we should simply inline the constants instead of adding this dependency. The communication is intentionally happening via extra credentials so as not to introduce any dependency or API, so adding a dependency here just to share a few strings seems wrong.